packages/terra-collapsible-menu-view/src/CollapsibleMenuView.jsx

```diff
@@ -45,6 +42,24 @@ const propTypes = {
    * Aligns the menu to the start of the container
    */
   isStartAligned: PropTypes.bool,
+
+  /**
+   * @private
+   * Switches to using horizontal ellipses instead of vertical ones when menu items are truncated.
+   */
+  useHorizontalIcon: PropTypes.bool,
+
+  /**
+   * @private
+   * Reverses the order of the rendered items.
+   */
+  isReversed: PropTypes.bool,
```

Comment:
I don't know that "isReversed" is clear enough for consumers of the API.  For example, which direction is the normal direction.  Do we need to make the property about the collapse direction?  Maybe that would be more clear.

Reply:
This is a private prop that is not intended to be used by consumers

Reply:
Even if it is private, I think is should be renamed for better readability and maintenance in the future.

Reply:
@cm9361 What do you think about `isReversedChildrenOrder`? I'm struggling to come up with a good name for it

Reply:
That name makes sense to me